src/ReductionHistory.tsx

```diff
@@ -292,6 +320,18 @@ function Row({ reduction, index }: { reduction: Reduction; index: number }): JSX
     return fileNameWithExtension.split('.')[0];
   };
 
+  const getFileTypeIcon = (fileName: string): JSX.Element => {
+    if (fileName.endsWith('.nxspe')) {
```

Comment:
The same comment I made about run files should be applied here, would it also be possible to move these to a utility file? Then it can be reused.